magma/t.py

```diff
@@ -98,17 +99,35 @@ def debug_name(self):
 
     def __le__(self, other):
         if not self.is_output():
-            self.wire(other)
```

Comment:
why does this need to change? (and below)

Reply:
hmm, that was left over from the old implementation (so it would dispatch to the logic in m.wire), will remove.